src/main/java/bio/terra/service/common/gcs/GcsUriUtils.java

```diff
@@ -13,7 +13,7 @@ public final class GcsUriUtils {
   /**
    * Parse a Google Cloud Storage URI into its component pieces
    *
-   * @param uri of type gs://<bucket_name>/<file_path_inside_bucket>
+   * @param uri of type gs://[bucket_name]/[file_path_inside_bucket]
```

Comment:
If you wanted to keep the `<` `>` you could instead wrap in `@code`
```suggestion
   * @param uri of type {@code gs://<bucket_name>/<file_path_inside_bucket>}
```

Reply:
Eh, yeah, I started with that and I honestly don't care about having brackets